dgv/monitoring/DAG-notification.py

```diff
@@ -183,6 +183,43 @@ def alert_if_awaiting_spam_comments():
         send_message(message, MATTERMOST_MODERATION_NOUVEAUTES)
 
 
+def alert_if_new_reports():
+    # DAG runs every 5min but if it fails we catch up with this variable
+    previous_report_check = Variable.get(
```

Comment:
Can't we fetch latest run instead of using variables?

Reply:
That's more complex, getting the latest run in which this specific task didn't fail. I can try if needed, but this should also work